modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -52,6 +53,11 @@ definition {
 			actionName = "${actionName}",
 			active = "${active}",
 			click = "save",
+			inputCondition = "${inputCondition}",
+			inputExpression = "${inputExpression}",
+			inputMethod = "${inputMethod}",
+			inputValue = "${inputValue}",
+			objectLabel = "${objectLabel}",
```

Comment:
What happens to other tests that are using this macro but don't have these values set? Will they still pass?

Reply:
Fixed

Reply:
What it seems to me is that `inputCondition` was being used as a variable to define a conditional in the macro. I modified it.